File: code/scenes/src/main/java/org/fbme/scenes/cells/SceneStyleAttributes.java
package org.fbme.scenes.cells;

import jetbrains.mps.editor.runtime.style.SimpleStyleAttribute;
import jetbrains.mps.openapi.editor.style.StyleAttribute;

import java.awt.*;

public class SceneStyleAttributes {

    public static final StyleAttribute<Color> SCENE_BACKGROUND = new SimpleStyleAttribute<>("scene-background");

    static {
        SCENE_BACKGROUND.register();
    }
}

File: code/scenes/src/main/kotlin/org/fbme/scenes/cells/SceneStyleAttributes.kt
package org.fbme.scenes.cells

import jetbrains.mps.editor.runtime.style.SimpleStyleAttribute
import jetbrains.mps.openapi.editor.style.StyleAttribute
import java.awt.Color

object SceneStyleAttributes {
    val SCENE_BACKGROUND: StyleAttribute<Color> = SimpleStyleAttribute("scene-background")

    init {
        SCENE_BACKGROUND.register()
    }
}
